presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
+* Fix query failure for certain join queries with "Equi criteria are empty,
```

Comment:
@rschlussel - this point isn't super clear to me - does it mean fixed the error whose message is this.

Reply:
yes, that's what I meant

Reply:
As Tim pointed out, can we talk about the real issue instead of the error message?